tests/unit/test_tiler.py

```diff
@@ -897,12 +887,7 @@ def it_can_scale_scores(self):
                         (0.8, CP(0, 10, 0, 10)),
                         (0.7, CP(0, 10, 0, 10)),
                         (0.5, CP(0, 10, 0, 10)),
-                    ],
-                    [
-                        (1.0, CP(0, 10, 0, 10)),
-                        (0.857142857142857, CP(0, 10, 0, 10)),
-                        (0.5714285714285714, CP(0, 10, 0, 10)),
-                    ],
```

Comment:
Removing this expectations produces a regression on the unit tests coverage. I would suggest to duplicate `it_can_calculate_highest_score_tiles` rename it as `it_can_calculate_highest_scaled_score_tiles` and try it using the `scaled=True` param when you call `highest_score_tiles = score_tiler._highest_score_tiles(slide)`

Reply:
UR right